src/Cake.Common/Tools/DotNetCore/Build/DotNetCoreBuildSettings.cs

```diff
@@ -42,11 +37,6 @@ public sealed class DotNetCoreBuildSettings : DotNetCoreSettings
         public string VersionSuffix { get; set; }
```

Comment:
We should add a string Version { get; set; } too, equivalent to
```csharp
 ArgumentCustomization = arg=>arg.Append(string.Concat("/p:Version=", semVersion)),
``` 

We should also add an 
```csharp
public IDictionary<string, IList<string>> Properties => _properties;
```
To support MSBuild properties without ArgumentCustomization, similar too [MSBuildSettings.Properties](https://github.com/cake-build/cake/blob/1eb28cf18637a1316c8f10327b9d70b9975507ea/src/Cake.Common/Tools/MSBuild/MSBuildSettings.cs#L32)

We should also add an 
```csharp
public ISet<string> Targets { get; }
```

Running `dotnet msbuild -h` looks like it support most MSBuild so future refactoring could be to see what we have common between the .NET Core CLI and MSBuild alias.